ui/apps/ui/src/app/pages/trainings-page/training-detail-page.component.html

```diff
@@ -26,84 +26,99 @@
+                  <p class="section" i18n>Resource URL type<br><span class="section-description">{{ myTraining?.url_type ?? '-' | filterPipe: 'url_type'  }}</span></p>
+                </div>
+                <div class="col-4">
+                  <p class="section" i18n>Authors<br>
+                    <ng-container *ngFor="let author of myTraining.author_names">
+                      <span class="section-description">{{ author}}, </span>
+                    </ng-container>
                   </p>
-                <p class="section" i18n>Expertise Level<br><span class="section-description">{{ myTraining.level_of_expertise }}</span></p>
-                <p class="section" i18n>Qualification<br><span class="section-description">{{ '-' }}</span></p>
-                <p class="section" i18n>License<br><span class="section-description">{{ myTraining.license }}</span></p>
+                </div>
               </div>
-              <div class="col-4">
-                <p class="section" i18n>Version Date<br><span class="section-description">{{ myTraining.publication_date | transformDatePipe }}</span></p>
-                <p class="section" i18n>Catalogue<br><span class="section-description">{{ myTraining.catalogue  }}</span></p>
-                <p class="section" i18n>Content Resource Type<br>
-                  <ng-container *ngFor="let resource of myTraining.resource_type">
-                    <span class="section-description">{{  resource }}</span><br>
-                  </ng-container>
-                </p>
-                <p class="section" i18n>Duration<br><span class="section-description">{{  myTraining.duration }}</span></p>
-               </div>
             </div>
 
-
+            <div class="details-box">
+              <div class="details-box-container">
+                <div class="svg-box">
+                  <div class="category-icon">
+                    <span class="helper"></span>
+                    <img [src]="'./assets/target.svg'" alt="">
+                  </div>
+                </div>
+                <p class="details-header" i18n>Learning Details</p>
+              </div>
+              <div  class="column-container">
+                <div class="col-4">
+                  <p class="section" i18n>Expertise Level<br><span class="section-description">{{ myTraining.level_of_expertise }}</span></p>
+                  <p class="section" i18n>Qualification<br><span class="section-description">{{ '-' }}</span></p>
+                  <p class="section" i18n>Content Resource Type<br>
+                    <ng-container *ngFor="let resource of myTraining.content_type">
+                      <span class="section-description">{{  resource }}</span><br>
+                    </ng-container>
+                  </p>
+                </div>
+                <div class="col-4">
+                  <p class="section" i18n>Duration<br><span class="section-description">{{  myTraining.duration }}</span></p>
```

Comment:
Duration can also be empty - so '-' will be useful here as well

Reply:
done